etc/helm/pachyderm/templates/pachd/config-job.yaml

```diff
@@ -37,13 +37,8 @@ spec:
               key: OAUTH_CLIENT_SECRET
         {{- end}}
         {{- if .Values.enterpriseServer.enabled }}
-        {{- if eq .Values.enterpriseServer.service.type "NodePort" }}
         - name: PACH_ADDR
-          value: "grpc://pach-enterprise:1650"
-        {{- else }}
-        - name: PACH_ADDR
-          value: "grpc://pach-enterprise:30650"        
-        {{- end }}
+          value: "grpc://pach-enterprise:31650"        
```

Comment:
make this port configurable

Reply:
also do we need to warn customers about ports changing? this change probably shouldn't go out till 2.2